templates/default/update-llnw.json.erb

```diff
@@ -1,51 +1,33 @@
 {
   "source_dir": "<%= @source_dir %>",
   "cdn_list": [
-    "llnw",
     "akamai",
     "edgecast",
     "cdnetworks"
```

Comment:
do we need `cdnetworks` since there is no cdnetworks block below?

Reply:
@komealy I was wondering the same but decided do not touch it. :)